src/test/unit/cli_unittest.cc

```diff
@@ -59,10 +60,19 @@ extern "C" {
     #include "sensors/battery.h"
     #include "sensors/gyro.h"
 
+    // rc/rx.h
+    float rcData[MAX_SUPPORTED_RC_CHANNEL_COUNT];
+
     void cliSet(const char *cmdName, char *cmdline);
     int cliGetSettingIndex(char *name, uint8_t length);
     void *cliGetValuePointer(const clivalue_t *value);
-    
+
+    void cliHelp(const char *, char *);
```

Comment:
This is terrible way to declare external functions. Changing function signature will silently break test in undefined way.
It is use in BF, but it is really bad idea.

IMO best way would be to add `<name>_uittest.h` and place declarations there + include it in `<name>.c` to get error checking (and forward declarations as a bonus)

But it is not directly related to this PR

Reply:
I think it is definitely a problem that should be solved in different task